src/screens/MainApp/Settings/index.js

```diff
@@ -4,9 +4,11 @@ import NavigationService from '~/navigationService'
 import * as firebase from 'firebase'
 import * as AnalyticsService from '~/analyticsService'
 import { goToPrivacyPolicyURL } from '~/mixins/privacyPolicyMixins'
+import MessagingService from '~/messagingService'
 
 const LogOut = async () => {
   try {
+    await MessagingService.deleteFCMTokenForCurrentUser()
     await firebase.auth().signOut()
     NavigationService.navigate('Login')
```

Comment:
If the FCM Token deletion fails, the user won't be signed out. I think it should be in a separate try/catch block, since we can safely ignore that error and continue with the logout flow.

Reply:
That function doesn't throw any error. Same as the one checking for permissions...